clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/view/EcsApplicationProvider.java

```diff
@@ -50,31 +51,32 @@ public EcsApplicationProvider(
 
   @Override
   public Application getApplication(String name) {
-
-    for (Application application : getApplications(true)) {
+    name = name.toLowerCase();
```

Comment:
This can have an impact below on casing, since it's lower case & application (though app should be lower case... just a caution of COULD be a bug someplace here).  IF going to lower case, should do name equals ignore case or similar.  KINDA concerned about the filtering as well on casing but not dug into that.

Reply:
Yeah it shouldnt be a problem. It is always coming as lower case as the caching agents are inserting the keys in lowercase. Similar to here in the AWS provider https://github.com/spinnaker/clouddriver/blob/master/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonApplicationProvider.groovy#L52

Reply:
Maybe both are setting it, but the caution flag is:  Don't trust that, as I've seen cases where SOMEHOW the data internally gets into mixed cases depending on the location.  I'd rather avoid any POTENTIAL bug in the future here.

Reply:
Reverting the change to not forcing a lowercase as the search previously was. I see what you are concerned about - we dont know how the ECS clusters/services are deployed (lower/uppercase etc) and how the application/moniker are configured. 
Lets keep the logic the same which so far didnt create any issues.